packages/skill-lesson/lib/products.ts

```diff
@@ -24,6 +24,32 @@ const productsQuery = groq`*[_type == "pricing"][0] {
   }
 }`
 
+const allProductsQuery = groq`*[_type == "pricing"] {
```

Comment:
shouldn't this be `_type == 'product'` instead of `pricing` ?

Reply:
nope